scripts/fast-reboot

```diff
@@ -667,14 +668,15 @@ fi
 set +e
 
 if [[ "$REBOOT_TYPE" = "fast-reboot" ]]; then
-    # Clear all routes except of default routes for faster reconciliation time.
-    sonic-db-cli APPL_DB eval "
-        for _, k in ipairs(redis.call('keys', '*')) do
-            if string.match(k, 'ROUTE_TABLE:') and not string.match(k, 'ROUTE_TABLE:0.0.0.0/0') and not string.match(k, 'ROUTE_TABLE:::/0') then \
-                redis.call('del', k)
-            end
-        end
-    " 0 > /dev/null
+    # Clear all routes except of default and connected routes for faster reconciliation time.
+    debug "Clearing routes..."
+    FILTER_ROUTES=0
+    python /usr/local/bin/fast-reboot-filter-routes.py || FILTER_ROUTES=$?
+    if [[ FILTER_ROUTES -ne 0 ]]; then
```

Comment:
1. This error handling is not added to 202205 branch PR. Any specific reason?
2. Again highlighting that this call is after `set +e` - so if filtering routes fail, device will still proceed to reboot. This was discussed earlier on 202205 PR and was acceptable to MLNX.